node/pkg/driver/executer/executer.go

```diff
@@ -64,10 +64,21 @@ func (e *Executer) ExecuteWithTimeout(mSeconds int, command string, args []strin
 	}
 
 	// If there's no context error, we know the command completed (or errored).
-	logger.Debugf("Output from command: %s", string(out))
 	if err != nil {
 		logger.Debugf("Non-zero exit code: %s", err)
 	}
+	return out, err
+}
+
+func (e *Executer) ExecuteWithTimeout(mSeconds int, command string, args []string) ([]byte, error) {
+	logger.Debugf("Executing command : {%v} with args : {%v}. and timeout : {%v} mseconds", command, args, mSeconds)
+
+	out, err := e.ExecuteWithTimeoutSilently(mSeconds, command, args)
+
+	outAsStr := string(out)
+	if strings.TrimSpace(outAsStr) != "" {
+		logger.Debugf("Output from command: %s", outAsStr)
+	}
 
 	logger.Debugf("Finished executing command")
```

Comment:
```suggestion
	outAsStr := string(out)
	if strings.TrimSpace(outAsStr) != "" {
		logger.Debugf("Output from command: %s", outAsStr)
		noOutputMessage := ""
	}
	else {
		noOutputMessage := " (no output)"
	}

	logger.Debugf("Finished executing command" + noOutputMessage)
```

Reply:
the all point is to reduce the log, if we are not expecting any return so why log it?

Reply:
what do you mean "not expecting"?
this just explicitly clarifies in the log that there was no output.
I don't think this significantly hampers the log reduction

Reply:
when we are talking about logs that are thousands of lines long I think that every line counts.

Reply:
this suggestion doesn't add more lines

Reply:
its adding to my change in case of empty respond from command

Reply:
yes, but as I wrote, the addition is insignificant (12 chars in an existing message, and only when there is no output), and makes the lack of output clear

Reply:
done